docs/machine-learning/how-to-guides/train-machine-learning-model-ml-net.md

```diff
@@ -0,0 +1,179 @@
+// Define Data Prep Estimator
+// 1. Concatenate Size and Historical into a single feature vector output to a new column called Features
+// 2. Normalize Features vector
+IEstimator<ITransformer> dataPrepEstimator =
+    mlContext.Transforms.Concatenate("Features", "Size", "HistoricalPrices")
+        .Append(mlContext.Transforms.NormalizeMinMax("Features"));
+
+// Create data prep transformer
+ITransformer dataPrepTransformer = dataPrepEstimator.Fit(trainData);
+
+// Apply tranforms to training data
+IDataView transformedTrainingData = dataPrepTransformer.Transform(trainData);
+```
+
+### Working with default column names
+
+Additionally, algorithms use default column names when none are specified. All trainers have a parameter called `featureColumnName` for the inputs of the algorithm and when applicable they also have a parameter for the expected value called `labelColumnName`. By default those values are `Features` and `Label` respectively. 
+
+By using the [`Concatenate`](xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate*) method during pre-processing to create a new column called `Features`, there is no need to specify the feature column name in the parameters of the algorithm since it already exists in the pre-processed `IDataView`. The label column is `CurrentPrice`, but since the [`ColumnName`](xref:Microsoft.ML.Data.ColumnNameAttribute.#ctor*) attribute is used in the data model, ML.NET renames the `CurrentPrice` column to `Label` which removes the need to provide the `labelColumnName` parameter to the machine learning algorithm estimator. 
+
+If you don't want to use the default column names, pass in the names of the feature and label columns as parameters when defining the machine learning algorithm estimator as demonstrated by the subsequent snippet:
+
+```csharp
+var UserDefinedColumnSdcaEstimator = mlContext.Regression.Trainers.Sdca(labelColumnName: "MyLabelColumnName", featureColumnName: "MyFeatureColumnName");
+```
+
+## Train the machine learning model
+
+Once the data is pre-processed, use the [`Fit`](xref:Microsoft.ML.Trainers.TrainerEstimatorBase`2.Fit*) method to train the machine learning model with the [`StochasticDualCoordinateAscent`](xref:Microsoft.ML.Trainers.SdcaRegressionTrainer) regression algorithm.
+
+```csharp
+// Define StochasticDualCoodrinateAscent regression algorithm estimator
+var sdcaEstimator = mlContext.Regression.Trainers.Sdca();
+
+// Build machine learning model
+var trainedModel = sdcaEstimator.Fit(transformedTrainingData);
+```
+
+## Extract model parameters
+
+After the model has been trained, extract the learned [`ModelParameters`](xref:Microsoft.ML.Trainers.ModelParametersBase`1) for inspection or re-training. The [`LinearRegressionModelParameters`](xref:Microsoft.ML.Trainers.LinearRegressionModelParameters) provide the bias and learned coefficients or weights of the trained model. 
+
+```csharp
+var trainedModelParameters = trainedModel.Model as LinearRegressionModelParameters;
```

Comment:
Do we want to clarify that this only works if the algorithm is the last transformation in the pipeline?

Reply:
In this case as well as the other samples I split out the pipelines so that you should always be able to extract the model parameters.